Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+            # Extract center body and duct parameterization parameters and Ducted fan design parameters
+            # Write them to self
+            # All inputs must be dictionaries
+            params_CB, params_duct, ducted_fan_design_params, case_name = args 
+            
+            self.centerbody_params: dict = params_CB
+            self.ducted_fan_design_params: dict = ducted_fan_design_params
+            self.duct_params: dict = params_duct
+            self.case_name: str = case_name
+                        
+
+        def GetGridSize(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `GetGridSize`

(ANN201)